src/styles/main.scss

```diff
@@ -0,0 +1,3 @@
+@import 'utilis/variables';
+@import 'utilis/base';
```

Comment:
Ensure that the file `utilis/base.scss` exists. This file might contain base styles or resets that are necessary for the project.